### PR TITLE
Edits/typo fixes

### PR DESCRIPTION
**Pull request by jeff-boykin**

### CODE REVIEW

File: curriculum/02-materials/lesson.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ creator:
 *After this lesson, you will be able to:*
 
 - Explain the field of data science, defining common roles, benefits, and trends.
-- Explore some popular tools and resources to visualize,  analyse, and model data.
-- Recognise the different types of problems that can be solved by data scientists.
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: Man! You are americanising everything! Leave us at least the dignity of "correct" spelling!

File: curriculum/02-materials/lesson.md [hunk 2]
```diff
@@ -114,7 +115,7 @@ Before we dive in, a bit about you!
 ### Guided Practice: Qualities of a data scientist
 
 - Programming skills
-- Maths and Stats knowledge
+- Math and statistics knowledge
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: Yuck! Its Maths for us as it is a MathematicS! Oh well...

File: curriculum/02-materials/lesson.md [hunk 3]
```diff
@@ -257,19 +258,19 @@ Libraries of code written to solve particular set of problems
 In Python there are many related packages relevant to data science: pandas, Scikit-learn, NumPy, etc. Some widely-used packages include
 
 * pandas
-	* Ever used Excel? How do yo fancy working with data structured in a similar way, but without the irritation of formatting, long formulae and better graphics. Well, use *pandas*
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: This one, please leave me this one... let's use the correct plural for formula -> formulae! Please?

File: curriculum/02-materials/lesson.md [hunk 4]
```diff
@@ -409,16 +410,16 @@ To get a hold of the starter code, you'll need to download these materials.
...
 
 *Algorithm*: A set of steps to accomplish a task.
 
 Algorithms need to have their steps in the right order.
 
 Think about an algorithm for getting dressed in the morning. 
 
-- What if you put on your coat before your jumper? 
-	- Your jumper would be on top of your coat and that would be silly! 
+- What if you put on your coat before your shirt?
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: Really? jumper is no good either? Blimey!

File: curriculum/02-materials/lesson.md [hunk 5]
```diff
@@ -446,15 +447,14 @@ Data In -> Change -> New Data Out
 
 > Instructor Note:
 > 
-> Divide the class in three groups and ask students to list the steps they follow to achieve an every-day task, e.g. make cheese on toast, commute to work, make a cup of tea.
-> Student need to break down steps into the smallest, discrete, sequential items etc
+> Divide the class in three groups and ask students to list the steps they follow to achieve an every-day task, e.g. making buttered toast, commuting to work, making a cup of coffee.
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: Cheese on toast is no  good? And tea is gone??? What??? Buttered toast is not quite the same... what about scones and cream? Americans!

File: curriculum/02-materials/lesson.md [hunk 6]
```diff
@@ -674,7 +674,7 @@ In this workshop, we've covered the following topics:
 Remember the skills and qualities of a data scientist?
 
 - Programming skills
-- Maths and Stats knowledge
+- Math and statistics knowledge
 - Business acumen (substantive expertise)
 - Plus: Communication skills
```
Comment (rogelj) [inline curriculum/02-materials/lesson.md]: All in all nothing major to revise, but I do feel like I should also change my accent... Any chance of leaving at least the references to the British things like cheese on toast or tea?

Comment (jeff-boykin) [inline curriculum/02-materials/lesson.md]: lol

File: curriculum/02-materials/lesson.md (hunk 5 above)
Comment (jeff-boykin) [inline curriculum/02-materials/lesson.md]: Haha last I checked, we only have one London office and like 10 US locations... but of course you're welcome to swap out terms when *you* teach it :)

**Pull request merged**